loops-and-arrays.js

```diff
@@ -30,7 +36,15 @@ const numbers = [2, 4, 5, 29, 38];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(numbers) geeft: [ 4, 8, 15, 87, 76 ];
```

Comment:
De logica om even getallen te verdubbelen en oneven getallen te verdrievoudigen is correct geïmplementeerd.